src/scss/constants.scss

```diff
@@ -141,6 +141,8 @@ $media-breakpoint--lg: 992px; // tablet--medium
 $media-breakpoint--xl: 1200px; // tablet--large
 $media-breakpoint--xxl: 1350px; // laptop
 
+$spacer: 1rem;
```

Comment:
Not necessary. There are `font-size--root(1)` and `font-size--parent(1)` functions used in many places.

Reply:
done ead67ab